README.md

```diff
@@ -182,19 +182,16 @@ Try this `npm.yaml` file:
 ```yaml
 repo:
   type: npm
+  path: /npm
```

Comment:
@brastak can we parse the path dynamically? It seems to be the same as repository name always, can we use it instead of config param?

Reply:
@g4s8 It's the question of component's responsibility. I think, it's better if the Pie class will know about repositories and the paths they handled. Then we will be able to introduces the changes in the single place. Currently, we have 2 path-related issues:
* In generally we handles flat repositories structure only (repo path is always single-level). It is bad for the case of personal repos in central.artipie.com - we need to use `user-repo-name` naming policy instead of more suitable `user/repo-name`. Single-level path requirement is the strange restriction and I'm not sure that we shall to handle it;
* Many of our adapters contain tests for the case when no context path exists at all and the repo handles root path (this case is impossible in our runtime).

I think, it's better to do the following:
- remove all paths from repo configs (not only NPM adapters containt it);
- pass actual repo path from Pie to SliceFromConfig and use it in adapter's constructor if they need this information.

But I think that this refactoring should be done as separate set of issues.

If you want, I can remove the `path` from here and will be always use the first part of URL, but it looks like that we are changing one temporary solution to another

Reply:
@brastak yes, it sounds reasonable. I think you can submit a ticket to propose `/user/repo` schema instead of `/repo`.